src/bindings/python/fluxacct/accounting/user_subcommands.py

```diff
@@ -151,7 +151,9 @@ def get_default_bank(cur, username):
     cur.execute(select_stmt, (username,))
     result = cur.fetchall()
 
-    return result[0][0]
+    if len(result) > 0:
```

Comment:
I think `if result:` is the Pythonic way of making this conditional (because a collection is `False` if it doesn't have any elements) but I'm kind of ambivalent about it so...

Reply:
Thanks @jameshcorbett, that's a good suggestion! I just force-pushed to use `if result:`. Will set MWP here